src/qt/forms/sendcoinsdialog.ui

```diff
@@ -760,10 +760,30 @@
            </layout>
           </item>
           <item>
+           <widget class="QLabel" name="fallbackFeeWarningLabel">
+            <property name="toolTip">
+             <string>Using the fallbackfee can result in sending a transaction that will take serval hours or days (or never) to confirm. Consider choosing your fee manually or wait until your have validated the complete chain.</string>
```

Comment:
I have though about that, but to not over-explain this, I was picking out the most important part, which is probably a validated chain. And since we persist the mempool, this is less of a problem.